web/src/views/DandisetLandingView/DandisetPublish.vue

```diff
@@ -75,17 +75,69 @@
                 </span>
               </v-col>
             </v-row>
-            <v-btn
-              block
-              :color="showPublishWarning ? 'error' : 'success'"
-              depressed
-              :disabled="publishButtonDisabled"
-              @click="publish"
+            <div
+              class="text-center"
             >
-              Publish
-              <v-spacer />
-              <v-icon>mdi-upload</v-icon>
-            </v-btn>
+              <v-dialog
+                v-model="showPublishChecklistDialog"
+                width="900"
+              >
+                <template #activator>
+                  <v-btn
+                    block
+                    :color="showPublishWarning ? 'error' : 'success'"
+                    depressed
+                    :disabled="publishButtonDisabled"
+                    @click="showPublishChecklistDialog = true"
+                  >
+                    Publish
+                    <v-spacer />
+                    <v-icon>mdi-upload</v-icon>
+                  </v-btn>
+                </template>
+
+                <v-card>
+                  <v-card-title class="text-h5 grey lighten-2">
+                    Publish Checklist
+                  </v-card-title>
+
+                  <v-card-text>
+                    <v-list>
+                      <span class="text-body-1 font-weight-bold">
+                        To maximize utility, have you checked the list of
+                        authors and keywords and have you added:
+                      </span>
+
+                      <v-list-item
+                        v-for="(item, i) in PUBLISH_CHECKLIST"
+                        :key="`checklist_item_${i}`"
+                        class="text-body-2 my-1"
+                        v-text="`${i+1}. ${item}`"
+                      />
+                    </v-list>
```

Comment:
Is there a way for these to contain HTML content, rather than just text? (This will benefit from being able to, e.g., italicize certain parts of the items.)

It seems like the HTML-native way to do this (if we really want a numbered list) is to just use an `<ol>` and let the renderer fill in the numbers. What's the vuetify way to do that?

Reply:
There doesn't appear to be a way to use `<ol>` in vuetify. You can leverage the `tag` prop to make the `v-list` and `v-list-item` elements use `<ol>` and `<li>` respectively, but it still doesn't render any numbers
![Screenshot from 2022-07-19 13-09-39](https://user-images.githubusercontent.com/37340715/179809492-bb75f116-f117-4682-a69f-22bf44bfe699.png)